### PR TITLE
Fix: Add tooltips and correct color contrast 

### PR DESCRIPTION
**Pull request by ElinorW**
## Overview

- Added tooltips to the 'minimize sidebar' and 'more actions' icons displayed to enhance keyboard navigation.

- Changed the color of the "patch" drop-down control to fit the color contrast ratio. 

### Demo

image

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
* Prefer bulleted description
* Start after checking out this branch
* Include any setup required, such as bundling scripts, restarting services, etc.
* Include test case, and expected output

### CODE REVIEW

**Comment by jobala** [comment]
I see two "more actions" tooltips when hovering over the gear icon.

image

File: src/app/views/app-sections/AppTitle.tsx [hunk 1]
```diff
@@ -9,12 +9,16 @@ export function appTitleDisplayOnFullScreen(
...
 
     return <div style={{ display: 'flex', width: '100%' }}>
-      <IconButton
-        iconProps={{ iconName: 'GlobalNavButton' }}
-        className={classes.sidebarToggle}
-        title='Minimise sidebar'
-        ariaLabel='Minimise sidebar'
-        onClick={() => toggleSidebar()} />
+      <TooltipHost
+        content='Minimise sidebar'
```
Comment (ddyett) [inline src/app/views/app-sections/AppTitle.tsx:13]: misspelt.  should be z

Comment (ddyett) [reply src/app/views/app-sections/AppTitle.tsx:13]: also what about localization here?

**Comment by jobala** [comment]
@ddyett @bettirosengugi @ElinorW @thewahome 

I am interested in knowing what you guys think about replacing the **settings** icon with a **context menu** icon. The settings icon is used for updating  user settings or profile in apps. The **context menu** icon is used for showing more actions. 

**Sample Context Menu Icon**
image

File: src/app/views/app-sections/AppTitle.tsx (hunk 1 above)
Comment (jobala) [reply src/app/views/app-sections/AppTitle.tsx:13]: > misspelt. should be z

@ddyett depends on which english pronunciation you're using :alien:

Comment (ddyett) [reply src/app/views/app-sections/AppTitle.tsx:13]: all the more reason for localization.  since the most popular page is en-US probably should be z.

**Comment by ddyett** [comment]
not sure how i feel about the icon.  there are some settings like actions and some more actions like actions. i'd have to think about it.

**Comment by jobala** [comment]
@ddyett these are a closer representation of how the context-menu icons look like in uifabric-react
image

**Comment by bettirosengugi** [comment]
> @ddyett these are a closer representation of how the context-menu icons look like in uifabric-react
> image

This is a good call out @jobala We got this feedback from Frederick as well. @ddyett may be we can have this as our first case for A/B testing?

**Pull request merged**